ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/utils/export/export.tsx

```diff
@@ -96,7 +96,7 @@ export const exportResultSet = async (
 ) => {
   return await fetch(`./internal/cql/transform/${transformer}`, {
     method: 'POST',
-    body: JSON.stringify(body),
+    body: JSON.stringify({ ...body, sorts: body.sorts || [] }),
```

Comment:
❓ I'm not sure if this is the best way to use a default value for sorts if sorts is empty in the body.

❓ I don't know how to add a test for this.

Reply:
📝  Now that I'm looking at this code in full, I'm wondering if it might be better to just make sort non-optional so everywhere we use it, it has to be passed in (even if we're just passing in []): 
```
      response = await exportResultSet('zipCompression', {
        searches,
        count,
        args: {
          transformerId: uriEncodedTransformerId,
        },
        sorts: []
      })
```

or we could change this function to construct the body here:
```
export const exportResultSet = async ({transformer, searches, count, sorts=[], args={}}: {
  transformer: string,
  searches: ResultSet[]
  count: number
  sorts?: Object[]
  args?: Object
}) => {
  return await fetch(`./internal/cql/transform/${transformer}`, {
    method: 'POST',
    body: JSON.stringify({
      searches,
      count,
      sorts,
      args
    }),
    headers: {
      'Content-Type': 'application/json',
    },
  })
}
```

Admittedly, this is a bit overkill though!

Reply:
Updated it to be required instead